packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -1039,13 +1039,22 @@ class StripeHelper {
       throw error.unknownCustomer(subscription.customer);
     }
 
+    let invoice = subscription.latest_invoice;
+    if (typeof invoice === 'string') {
+      // if we have to do a fetch, go ahead and ensure we also get the additional needed resource
+      invoice = await this.stripe.invoices.retrieve(invoice, {
+        expand: ['charge'],
```

Comment:
Should this be `charges`? Or use the `CHARGES_RESOURCE` value?

Reply:
I think this is different - [`charge` is a property of an invoice resource](https://stripe.com/docs/api/invoices/object#invoice_object-charge), whereas "charges" is a standalone resource.